content/W2015/W15-event-grad-info-session.md

```diff
@@ -0,0 +1,16 @@
+Title: Graduate info Session
+Date: 2015-02-04 05:30pm
+Category: Events
+Tags: soical, info-session, research
```

Comment:
"soical" is a typo. I would use "academia" rather than research as a tag. Trying to figure out the last tag... this is somewhere between a talk and a panel.